cookbook/recipes/default.rb

```diff
@@ -0,0 +1,105 @@
+#
+# Cookbook Name:: warden
+# Recipe:: default
+#
+# Copyright (C) 2016 Rapid7 LLC.
+#
+# Distributed under terms of the MIT License. All rights not explicitly granted
+# in the MIT license are reserved. See the included LICENSE file for more details.
+#
+
+#######################
+## Install NodeJS 4.x
+#
+# This should be moved into a shared cookbook
+##
+include_recipe 'apt::default'
+
+apt_repository 'nodejs-4x' do
+  uri 'https://deb.nodesource.com/node_4.x'
+  distribution node['lsb']['codename']
+  components ['main']
+  key 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key'
+end
+
+package 'nodejs'
+#######################
+
+node.default['warden']['version'] = cookbook_version
+
+group node['warden']['group'] do
+  system true
+end
+
+user node['warden']['user'] do
+  comment 'warden operator'
+  system true
+
+  gid node['warden']['group']
+  home node['warden']['paths']['directory']
+end
+
+directory node['warden']['paths']['directory'] do
+  owner node['warden']['user']
+  group node['warden']['group']
+  mode '0755'
+
+  recursive true
+end
+
+## Fetch and install warden
+remote_file 'warden' do
+  source warden::Helpers.github_download('rapid7', 'warden', node['warden']['version'])
```

Comment:
Should be `Warden::Helpers` once you change the module name in `cookbook/libraries/helpers.rb`.